model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -175,6 +176,72 @@ message PTransform {
   DisplayData display_data = 6;
 }
 
+message StandardPTransformUrns {
+  enum Enum {
+    // Represents Beam's parallel do operation.
+    // Payload: ParDoPayload.
+    // TODO(BEAM-3595): Change this to beam:transform:pardo:v1.
+    PAR_DO = 0 [(standard_urn) = "urn:beam:transform:pardo:v1"];
+
+    // Represents Beam's flatten operation.
+    // Payload: None.
+    FLATTEN = 1 [(standard_urn) = "beam:transform:flatten:v1"];
+
+    // Represents Beam's group-by-key operation.
+    // Payload: None
+    GROUP_BY_KEY = 2 [(standard_urn) = "beam:transform:group_by_key:v1"];
+
+    // Represents the operation generating a single empty element.
+    IMPULSE = 3 [(standard_urn) = "beam:transform:impulse:v1"];
+
+    // Represents the operation to read a Bounded or Unbounded source.
+    // Payload: ReadPayload.
+    READ = 4 [(standard_urn) = "beam:transform:read:v1"];
+
+    // Represents the Window.into() operation.
+    // Payload: WindowIntoPayload.
+    ASSIGN_WINDOWS = 5 [(standard_urn) = "beam:transform:window_into:v1"];
+
+    // Represents the TestStream.
+    // Payload: TestStreamPayload
+    TEST_STREAM = 6 [(standard_urn) = "urn:beam:transform:teststream:v1"];
+
+    // Represents the Combine.perKey() operation.
+    // If this is produced by an SDK, it is assumed that the SDK understands pgbkcv,
+    // merge_accumulators and extract_outputs.
+    // Payload: CombinePayload
+    COMBINE_PER_KEY = 7 [(standard_urn) = "beam:transform:combine_per_key:v1"];
+
+    // Represents the Combine.globally() operation.
+    // If this is produced by an SDK, it is assumed that the SDK understands pgbkcv,
+    // merge_accumulators and extract_outputs.
+    // Payload: CombinePayload
+    COMBINE_GLOBALLY = 8 [(standard_urn) = "beam:transform:combine_globally:v1"];
+
+    // Represents the Combine.groupedValues() operation.
+    // If this is produced by an SDK, it is assumed that the SDK understands pgbkcv,
+    // merge_accumulators and extract_outputs.
+    // Payload: CombinePayload
+    COMBINE_GROUPED_VALUES = 9 [(standard_urn) = "beam:transform:combine_grouped_values:v1"];
+
+    COMBINE_PGBKCV = 10 [(standard_urn) = "beam:transform:combine_pgbkcv:v1"];
+    COMBINE_MERGE_ACCUMULATORS = 11 [(standard_urn) = "beam:transform:combine_merge_accumulators:v1"];
+    COMBINE_EXTRACT_OUTPUTS = 12 [(standard_urn) = "beam:transform:combine_extract_outputs:v1"];
```

Comment:
Separate fn api-only URNs from pipeline-level URNs

Reply:
Disagree here, much better to document all the combine URNs together than some in one place and some in another.

Reply:
I can accept this. They should live perhaps in an enum inside or alongside the `Combine` payload.

As you know, I broadly believe there is value is being clear about what is a PCollection(s)-to-PCollection(s) transformation versus what is an instruction that we can tell an SDK harness to apply to a bundle. Losing that distinction has already led to proposals that things that used to be automatic - like combiner lifting - being pushed back into the submitted pipeline.

So my only issue is I can't really tell from this PR if we are taking the position that these are all valid PCollection(s)-to-PCollection(s) transformations that just happen to only be used as instructions, or if we are just not being clear about the distinction.